### ISSUE TITLE
Exception with custom numbering using list of strings

### ISSUE THREAD

**Issue by mitchpaulus** [lierdakil/pandoc-crossref#226]
I am attempting to use a custom numbering scheme and continue to run into the following exception:

```
pandoc-crossref: Expected string in metadata field tblLabels but got list
CallStack (from HasCallStack):
  error, called at lib/Text/Pandoc/CrossRef/Util/Meta.hs:69:25 in pandoc-crossref-0.3.4.0-8iRIelU3HRWKXVIDtAHQsV:Text.Pandoc.CrossRef.Util.Meta
Error running filter pandoc-crossref:
Filter returned error status 1
```

I tried different variations of specifying the yaml array and I got the exception with the Windows binary and after building from source. I also tried with figLabels and that was the same. 

Here's a MWE of what is failing, trying to run `pandoc --filter pandoc-crossref -o test.docx test.md`:

```markdown
---
tblTitle: "My Table"
tblLabels: ["first", "second"]
---

This is a table

Table: My Caption {#tbl:uniqTableRef}

Field 1 | Field 2
--------|---------
1       | 2

```
The `arabic`, `roman`, and `alpha x` options all worked. The documentation was a little strange with the string `yaml` at the beginning as well when it says

For example, `yaml figLabels: [α, β, γ, 1, 2, 3, I, II, III]`

Am I missing something from the documentation?

Appreciate any help.

**Comment by lierdakil** [comment]
That's a bug. Thanks for reporting! Should be fixed in [v0.3.4.1](https://github.com/lierdakil/pandoc-crossref/releases/tag/v0.3.4.1). Binaries should appear in a couple hours.

**Comment by mitchpaulus** [comment]
@lierdakil I confirmed that v0.3.4.1 fixed the issue for me. Greatly appreciate the quick fix, and thanks for maintaining this project! I'll let you close this issue down when you're ready.
